lib/internal/bootstrap_node.js

```diff
@@ -307,6 +308,24 @@
     return wrappedConsole;
   }
 
+  function setupInspectorCommandLineAPI() {
+    const inspector = process.binding('inspector');
+    const addCommandLineAPIMethod = inspector.addCommandLineAPIMethod;
+    if (!addCommandLineAPIMethod) return;
+
+    const Module = NativeModule.require('module');
+    const path = NativeModule.require('path');
+    const cwd = tryGetCwd(path);
+
+    const consoleAPIModule = new Module('[consoleAPI]');
+    consoleAPIModule.filename = path.join(cwd, consoleAPIModule.id);
+    consoleAPIModule.paths = Module._nodeModulePaths(cwd);
+
+    addCommandLineAPIMethod('require', function require(request) {
+      return consoleAPIModule.require(request);
+    });
```

Comment:
The REPL does:

```js
  var module = new Module('<repl>');
  module.paths = Module._resolveLookupPaths('<repl>', parentModule, true) || [];

  var require = internalModule.makeRequireFunction(module);
  context.module = module;
  context.require = require;
```

`_resolveLookupPaths` combines `Module._nodeModulePaths()` and `Module.globalPaths` (containing things like `~/.node_modules` and `$NODE_PATH`). We obviously don't have a `parentModule` here, but we can emulate `_resolveLookupPaths` by simply concatenating the two arrays.

`makeRequireFunction` adds things like `require.resolve`, `.main`, `.extensions`, and `.cache`, which is good for consistency as well.

Also `[consoleAPI]` isn't really useful to the user, so `<inspector console>`?

Another thing is `module`. In the REPL, `module` is available as a variable (as seen in `context.module = module` above). Such should be the case here as well if possible.